packages/components/src/button/stories/best-practices.mdx

```diff
@@ -0,0 +1,65 @@
+import { Meta } from '@storybook/blocks';
+import FigmaEmbed from '/storybook/components/figma-embed';
+
+<Meta title="Components/Actions/Button/Best Practices" />
+
+# Button
+Buttons allow users to take actions and make selections with a single click or tap.
+
+<FigmaEmbed url="https://www.figma.com/design/V7NfxdP0ybEMXf01WyWGKB/Storybook-documentation-Figma-assets?node-id=5-280&node-type=frame&t=JLKM1grrtWleRkFR-11" title="kitchen-sink-button" height="400" />
```

Comment:
@auareyou I'm wondering if we can set up a better frame size for the Figma embeds, so the content isn't needlessly scaled down when rendered in the iframe. The max content width for the Storybook doc page is 1000px, while the Figma frames (for example [this](https://www.figma.com/design/V7NfxdP0ybEMXf01WyWGKB/Storybook-documentation-Figma-assets?node-id=5-640&t=H04Yc7wXcV8QE4KD-4)) are 1280px wide. Does the initial zoom factor become closer to 100% when the Figma frame widths are reduced? (I would love to just set a fixed 100% zoom factor from the code side, but I don't think there's a way to do that.)

Reply:
Yes, @mirka, we can definitely change the height. Are you suggesting changing the width to 1000px in the images and then whatever height is proportional?

Reply:
> Are you suggesting changing the width to 1000px in the images and then whatever height is proportional?

Right. But I don't know whether the height proportionality affects the initial zoom factor, and I don't have edit access for Figma, so could you experiment a bit to see if there is a better size setup? It's ok if nothing works, but just wanted to check if there's an easy way to make it closer to 100%.

Reply:
I've resized them to 1000px by 300px. Let me know if it doesn't work.

Reply:
Did you experiment with a couple values? It's a lot closer to 100%, but still slightly off (the buttons are 36px high instead of 40px), no? Seems like something we can get right once, and reuse those metrics for everything.